lib/src/model/user.dart

```diff
@@ -223,16 +220,28 @@ class UserLinks extends ModelBase {
     };
   }
 
-  factory UserLinks.fromJson(Map<String, dynamic> json) {
+  factory UserLinks.fromJson(Map<String, dynamic>? json) {
```

Comment:
Have you seen responses does not have any user links, or are you trying to be defensive? I would rather have the models reflect that a field might not be available by making it nullable. But only if it actually is optional.